sdk/sphere/Azure.ResourceManager.Sphere/api/Azure.ResourceManager.Sphere.netstandard2.0.cs

```diff
@@ -0,0 +1,611 @@
+        public static Azure.ResourceManager.Sphere.Models.ImageType NormalWorldDtb { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.ImageType NormalWorldKernel { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.ImageType NormalWorldLoader { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.ImageType Nwfs { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.ImageType OneBl { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.ImageType Other { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.ImageType PlutonRuntime { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.ImageType Policy { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.ImageType RecoveryManifest { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.ImageType RootFs { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.ImageType SecurityMonitor { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.ImageType Services { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.ImageType TrustedKeystore { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.ImageType UpdateCertStore { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.ImageType WifiFirmware { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Sphere.Models.ImageType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Sphere.Models.ImageType left, Azure.ResourceManager.Sphere.Models.ImageType right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Sphere.Models.ImageType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Sphere.Models.ImageType left, Azure.ResourceManager.Sphere.Models.ImageType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class ListDeviceGroupsContent
+    {
+        public ListDeviceGroupsContent() { }
+        public string DeviceGroupName { get { throw null; } set { } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct OSFeedType : System.IEquatable<Azure.ResourceManager.Sphere.Models.OSFeedType>
```

Comment:
This name is quite generic, could we rename it to be more specific? For instance, maybe simply `SphereOSFeedType`?

Reply:
fixed in latest commit